### ISSUE TITLE
[Bug]: Default Commander only has Team Leader perks

### ISSUE THREAD

**Issue by Grusey1** [igorkis-scrts/A3-Antistasi-Plus#128]
### Describe the bug

Default Commander used to be kind of a "god" role that had all the skills (medic, engineer etc.). It seems that it only has Team Leader perks now. I'm not sure if this is intentional behavior, if so then apologise for unnecessary bug report.

### How to reproduce

Going in game in the default commander slot.

### Version

2.1.2 and 2.1.3

### Have you altered the code?

No

### Map

Takistan

### What server?

Private dedicated server with Headless client(s)

### Mods

```html
-mod=@3CB_BAF_Equipment;@3CB_BAF_Vehicles;@3CB_BAF_Weapons;@ACRE2;@CBA_A3;@Sling_Load_Rigging;@twc_framework;@3den_Enhanced;@3CB_BAF_Units;@3CB_Factions;@3CB_Weapons_ACE_Compat;@ace;@ACE_Compat__RHS_AFRF;@ACE_Compat__RHS_GREF;@ACE_Compat__RHS_SAF;@ACE_Compat__RHS_USAF;@Achilles;@CUP_Terrains__Core;@Gruppe_Adler_Trenches;@Immersion_Cigs;@Improved_Craters;@Prone_Launcher;@Rat_works__L2A1_Antistructure_Launcher;@RHSAFRF;@RHSGREF;@RHSSAF;@RHSUSAF;@TWC_2_Inch_Mortar;@Simple_Armbands;@RKSL_Attachments_v302;@EM;@BloodLust;@CUP_Terrains__Maps;@JOINT_WITH_WEED;@Simple_Suppress;@Antistasi_Plus;
```

**Comment by igorkis-scrts** [comment]
I see a lot of untested and straight up unsupported mods. 
Try to load Antistasi Plus only and see if issue persists, because it works as intended in supported environment (commander has all the skills he should have).

**Comment by igorkis-scrts** [comment]
Fixed in 2.1.4.
